README.rst

```diff
@@ -42,28 +42,51 @@ If you wan to run unit tests:
 Specifications
 --------------
 
-Point Cloud
-~~~~~~~~~~~
-
-Point Cloud Tile Format:
-https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/PointCloud
-
-.. image:: https://raw.githubusercontent.com/Oslandia/py3dtiles/master/docs/pc_layout.png
-    :width: 700px
-    :align: center
+Generic Tile
+~~~~~~~~~~~~
 
 The py3dtiles module provides some classes to fit into the
 specification:
 
 - *Tile* with a header *TileHeader* and a body *TileBody*
-- *TileHeader* represents the first 28 bytes (magic value, version, ...)
-- *TileBody* contains the feature table *FeatureTable* and the batch table (not supported for now)
-- *FeatureTable* with a header *FeatureTableHeader* and a *FeatureTableBody*
-- *FeatureTableBody* which contains features of type *Feature*
+- *TileHeader* represents the metadata of the tile (magic value, version, ...)
+- *TileBody* contains varying semantic and geometric data depending on the the tile's type
 
-Moreover, a utility class *TileReader* is available to read a *.pnts*
+Moreover, a utility class *TileReader* is available to read a tile
 file as well as a simple command line tool to retrieve basic information
-about a Point Cloud file **py3dtiles\_info**.
+about the tile **py3dtiles\_info**.
+
+**How to use py3dtiles\_info**
+
+If we want to retrieve basic information about the file *pointCloudRGB.pnts*:
+
+.. code-block:: shell
+    $ py3dtiles_info tests/pointCloudRGB.pnts
+    Tile Header
+    -----------
+    Magic Value:  pnts
+    Version:  1
+    Tile byte length:  15176
+    Feature table json byte length:  148
+    Feature table bin byte length:  15000
+
+    Feature Table Header
+    --------------------
+    {'POSITION': {'byteOffset': 0}, 'RGB': {'byteOffset': 12000}, 'POINTS_LENGTH': 1000, 'RTC_CENTER': [1215012.8828876738, -4736313.051199594, 4081605.22126042]}
+
+    First point
+    -----------
+    {'Z': -0.17107764, 'Red': 44, 'X': 2.19396, 'Y': 4.4896851, 'Green': 243, 'Blue': 209}
```

Comment:
FWIW, I used the code block syntax in order to correctly render the README on pypi (I think Markdoiwn is not supported).